# Defines the discretized states of Hyq with joint angles

RobotStateJoints[]  states        # trajectory of states
float64             t_start       # the global time of the first state